src/styles/utilities/spacing.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Spacing utility styles with Bootstrap inspired naming.
+ * All styles should be incremented by units of 4.
+ *
+ * https://getbootstrap.com/docs/5.0/utilities/spacing
+ */
+export default {
+    m0: {
+        margin: 0,
+    },
+
+    mr0: {
+        marginRight: 0,
+    },
+
+    mr1: {
+        marginRight: 4,
+    },
+
+    mr2: {
+        marginRight: 8,
+    },
+
+    mr3: {
+        marginRight: 12,
+    },
+
+    ml1: {
+        marginLeft: 4,
+    },
+
+    ml2: {
+        marginLeft: 8,
+    },
+
+    mt1: {
+        marginTop: 4,
+    },
+
+    mt2: {
+        marginTop: 8,
+    },
+
+    mt3: {
+        marginTop: 12,
+    },
+
+    mt4: {
+        marginTop: 16,
+    },
+
+    mt5: {
+        marginTop: 20,
+    },
+
+    mt6: {
+        marginTop: 24,
+    },
+
+    mb1: {
+        marginBottom: 4,
+    },
+
+    mb2: {
+        marginBottom: 8,
+    },
+
+    mb3: {
+        marginBottom: 12,
+    },
+
+    mb4: {
+        marginBottom: 16,
+    },
+
+    mb5: {
+        marginBottom: 20,
+    },
+
+    mb6: {
+        marginBottom: 24,
+    },
+
+    mbn1: {
```

Comment:
this name feels a bit cryptic to me. I wonder if `mb1neg` makes more sense? I don't feel strongly at all.

Reply:
Yeah we can make it whatever. I thought it would be interesting to use the same naming conventions that Bootstrap uses for these.

Reply:
Ah cool, I'm totally down to follow suit with Bootstrap.

Reply:
Yeah, I think that's a good convention to follow.